src/components/onboarding/steps/budget.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react"
+import styled from "styled-components"
+
+import SelectableToggle from "../selectable_toggle"
+import { StepProps } from "../types"
+import { Layout } from "./layout"
+
+const OptionsContainer = styled.div`
+width: 450px;
+margin: 0 auto 100px;
+&:last-child {
+  border-bottom: 1px solid #e5e5e5;
+}
+`
+
+interface State {
+  selection: string
```

Comment:
Nit: It’s generally less brittle to use index based selections than comparing captions. It’s unlikely that we’ll do this any time soon, but for instance consider that we’d ever want to localise the copy, then this would need to be reworked to take multiple languages into account, whereas indices would be stable.

Reply:
Not quite sure I understand what you mean here? Isn't the text being produced based on what the index of the options array is?

Reply:
Yes it is _now_, but in general comparing indices should be favoured over comparing titles because those are UI level details, whereas indices are logic level details.